templates/docs/examples/patterns/tiered-list/50-50-desktop-with-description-cta.html

```diff
@@ -0,0 +1,89 @@
+{% extends "_layouts/examples.html" %}
+{% from "_macros/tiered-list.jinja" import tiered_list %}
+
+{% block title %}Tiered list / 50/50 on desktop with description CTA{% endblock %}
+
+{% block standalone_css %}patterns_all{% endblock %}
+
+{% block content %}
+{%- call(slot) tiered_list(is_description_full_width_on_desktop=false, is_list_full_width_on_tablet=true) -%}
+  {%- if slot == 'title' -%}
+    <h2>H2 - up to two lines; ideally one.</h2>
+  {%- endif -%}
+
+  {%- if slot == 'description' -%}
+    <p>A <a href="#">private cloud</a> provides organisations with on-demand
+    compute, storage and other resources that can be accessed over the network
+    and that are reserved exclusively for them - either in their own data centre
+    or via a 3rd party.</p>
+
+    <!-- Optional CTA block -->
+    <div class="p-cta-block">
```

Comment:
```suggestion
    <div class="p-cta-block is-borderless">
```

The default CTA block has a top border that uses the same color as the muted rules that separate the list elements. Maybe we should use the borderless CTA block variant to make this more clear, but I'll defer to Design on this.
@lyubomir-popov 

<img width="778" alt="Screenshot 2024-08-21 at 4 52 00 PM" src="https://github.com/user-attachments/assets/d74c08d7-0cd3-4197-9ab2-eacbb05f7468">

Reply:
The CTA blocks appear with this same muted border as the list elements in Figma, so this follows Design spec. It just stood out to me, so I figured I'd flag it